docs/getting-started-guides/fedora/flannel_multi_node_cluster.md

```diff
@@ -0,0 +1,219 @@
+        inet6 ::1  prefixlen 128  scopeid 0x10<host>
+        loop  txqueuelen 0  (Local Loopback)
+        RX packets 0  bytes 0 (0.0 B)
+        RX errors 0  dropped 0  overruns 0  frame 0
+        TX packets 0  bytes 0 (0.0 B)
+        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
+```
+
+* From any node in the cluster, check the cluster members by issuing a query to etcd server via curl. If you set up a 1 master and 3 nodes cluster, you should see one block for each node showing the subnets they have been assigned. You can associate those subnets to each node by the MAC address (VtepMAC) and IP address (Public IP) that is listed in the output.
+
+```
+# curl -L http://fed-master:4001/v2/keys/coreos.com/network/subnets | python -mjson.tool
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100   949  100   949    0     0   107k      0 --:--:-- --:--:-- --:--:--  115k
+{
+    "action": "get",
+    "node": {
+        "createdIndex": 9,
+        "dir": true,
+        "key": "/coreos.com/network/subnets",
+        "modifiedIndex": 9,
+        "nodes": [
+            {
+                "createdIndex": 156957,
+                "expiration": "2015-04-25T16:18:57.165341389Z",
+                "key": "/coreos.com/network/subnets/18.16.90.0-24",
+                "modifiedIndex": 156957,
+                "ttl": 69821,
+                "value": "{\"PublicIP\":\"192.168.122.127\",\"BackendType\":\"vxlan\",\"BackendData\":{\"VtepMAC\":\"b2:17:21:77:ce:62\"}}"
+            },
+            {
+                "createdIndex": 156961,
+                "expiration": "2015-04-25T16:19:00.51742357Z",
+                "key": "/coreos.com/network/subnets/18.16.83.0-24",
+                "modifiedIndex": 156961,
+                "ttl": 69825,
+                "value": "{\"PublicIP\":\"192.168.122.36\",\"BackendType\":\"vxlan\",\"BackendData\":{\"VtepMAC\":\"32:24:29:48:22:ae\"}}"
+            },
+            {
+                "createdIndex": 156954,
+                "expiration": "2015-04-25T16:18:54.132629408Z",
+                "key": "/coreos.com/network/subnets/18.16.29.0-24",
+                "modifiedIndex": 156954,
+                "ttl": 69818,
+                "value": "{\"PublicIP\":\"192.168.122.77\",\"BackendType\":\"vxlan\",\"BackendData\":{\"VtepMAC\":\"e6:c8:c9:d7:8e:df\"}}"
+            }
+        ]
+    }
+}
+
+```
+
+* From all nodes, review the `/run/flannel/subnet.env` file.  This file was generated automatically by flannel.
+
+```
+# cat /run/flannel/subnet.env
+FLANNEL_SUBNET=18.16.29.1/24
+FLANNEL_MTU=1450
+FLANNEL_IPMASQ=false
+```
+
+* At this point the cluster is set up and we can test it. We have etcd running on the kubernetes master, and flannel / docker running on kubernetes nodes. Next steps are for testing cross-host container communication which will confirm that docker and flannel are configured properly.
```

Comment:
it would good to mention starting docker and any configuration needed (should be none as of f22+), and that docker needs to be started after flannel.